initializr-web/src/main/java/io/spring/initializr/web/project/ProjectRequest.java

```diff
@@ -175,4 +175,11 @@ public void setBaseDir(String baseDir) {
 		this.baseDir = baseDir;
 	}
 
+	private String cleanInputValue(String inputString) {
+		if (StringUtils.hasText(inputString)) {
+			return org.apache.commons.lang3.StringUtils.stripAccents(inputString);
```

Comment:
I wonder if we should do the simple thing for now (as described in the article in the issue). We can expand if necessary, rather than calling third-party code.

Reply:
As it was already in dependency, so thought we can use it. But yes we can have more control if we use over methods.